src/lib/integration_test_cloud_engine/stack_driver_log_engine.ml

```diff
@@ -40,7 +40,7 @@ let log_filter_of_event_type =
       (* TODO change this condition to Event_type Framework t (ie events from the Puppeteer), and then the following condition is Event_type Daemon t (ie events from the mina daemon, which is most of them) *)
   | Event_type Node_offline ->
```

Comment:
Now that we have the new interface for describing event type parser inputs, I think we can write this function in a better way than this. You should be able to dynamically compute these filters from the the event type module's `event_parser` value.